src/test/resources/data/ac_test/modifiers/advancements/balanced_diet.json

```diff
@@ -7,6 +7,11 @@
 	},
 	{
 	  "type": "rewards",
+	  "conditions": [
+		  {
+			  "type": "forge:true"
+		  }
+	  ],
```

Comment:
If the conditions aren't needed, why even put them in the JSON?
Although, a new modifier to use as a test for a condition would be nice.

Reply:
It was just to show the syntax, but it doesn't really need to be there